README.md

```diff
@@ -6,7 +6,7 @@
 
 Fork Awesome is a full suite of 705 pictographic icons for easy scalable vector graphics on websites, originally created by [Dave Gandy](https://twitter.com/davegandy) and now maintained by a community.
 
-Following concerns regarding [the development of Font Awesome](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956), the PR Freeze since Oct 2016 and the direction Fort Awesome is taking with the version 5.0 of their project, we are forking Font Awesome (4.7), in order to build on this incredible tool Dave Gandy has given us, while at the same time allowing this project to be run by a distributed community of contributors.
+Following concerns regarding [the development of Font Awesome](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956), the PR Freeze since Oct 2016 and the direction Font Awesome is taking with the version 5.0 of their project, we are forking Font Awesome (4.7), in order to build on this incredible tool Dave Gandy has given us, while at the same time allowing this project to be run by a distributed community of contributors.
```

Comment:
Actually, this is not a mistake. Maybe it could be rephrased differently. But FoRt Awesome is the "company" behind FoNt Awesome. So in this phrase, we speak about a company decision with the release of the version 5.0 of their project. But if you'd like to rephrase this so that it waives the confusion, please make a proposition.

Reply:
Maybe instead of ``Fort Awesome`` use ``[Fort Awesome](https://fortawesome.com/)`` or ``<abbr title="Fort Awesome is the company behind Font Awesome">Fort Awesome</abbr>``

Reply:
I like the first idea...